### PR TITLE
[CLC-397]: Add DMT Command Documentation

### PR DESCRIPTION
**Pull request by kutluhanmetin**

### CODE REVIEW

File: docs/modules/migrate/pages/dmt-command-reference.adoc [hunk 1]
```diff
@@ -0,0 +1,90 @@
+= dmt
```
Comment (rebekah-lawrence) [inline docs/modules/migrate/pages/dmt-command-reference.adoc:1]: DMT Command Reference

File: docs/modules/migrate/pages/dmt-command-reference.adoc [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+
+dmt commands are a group of migration operations.
```
Comment (rebekah-lawrence) [inline docs/modules/migrate/pages/dmt-command-reference.adoc:3]: The DMT commands allow you to manage your migration.

The DMT commands have the following format:

File: docs/modules/migrate/pages/dmt-command-reference.adoc [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+== Commands
+
+* <<dmt start, dmt start>>
+* <<dmt status, dmt status>>
+* <<dmt cancel, dmt cancel>>
+* <<dmt estimate, dmt estimate>>
+
+== dmt start
+
+Starts a new migration with the given configuration.
```
Comment (rebekah-lawrence) [inline docs/modules/migrate/pages/dmt-command-reference.adoc:19]: specified configuration

File: docs/modules/migrate/pages/dmt-command-reference.adoc [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+|Parameter|Required|Description|Default
+
+|`--output-dir`, `-o`
+|Optional
+|Output directory for the migration report.
+|Current working directory
+
+|`--yes`
+|Optional
+|Skip confirming the migration operation.
```
Comment (rebekah-lawrence) [inline docs/modules/migrate/pages/dmt-command-reference.adoc:41]: If set, the migration operation is not confirmed.

File: docs/modules/migrate/pages/dmt-command-reference.adoc [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+|`false`
+
+|===
+
+== dmt status
+
+Shows the status of the currently running migration.
```
Comment (rebekah-lawrence) [inline docs/modules/migrate/pages/dmt-command-reference.adoc:48]: Displays the status...

File: docs/modules/migrate/pages/dmt-command-reference.adoc [hunk 6]
```diff
@@ -0,0 +1,90 @@
...
+|`--output-dir`, `-o`
+|Optional
+|Output directory for the migration report.
+|Current working directory
+
+|===
+
+== dmt cancel
+
+Cancels the currently running migration.
```
Comment (rebekah-lawrence) [inline docs/modules/migrate/pages/dmt-command-reference.adoc:72]: Cancels the migration.

(you can't cancel one that isn't running, and you can only run one at a time, so it isn't necessary to explain that it is the currently-running migration)

File: docs/modules/migrate/pages/dmt-command-reference.adoc [hunk 7]
```diff
@@ -0,0 +1,90 @@
...
+Usage:
+
+[source,bash]
+----
+dmt cancel
+----
+
+== dmt estimate
+
+Estimates the migration time with the given configuration.
```
Comment (rebekah-lawrence) [inline docs/modules/migrate/pages/dmt-command-reference.adoc:83]: Estimates the duration of the migration using the specified configuration.

File: docs/modules/migrate/partials/nav.adoc [hunk 8]
```diff
@@ -1,5 +1,6 @@
 ** xref:deploy:versioning-compatibility.adoc[Versioning and Compatibility]
 ** xref:migrate:data-migration-tool.adoc[]
+** xref:migrate:dmt-command-reference.adoc[]
```
Comment (rebekah-lawrence) [inline docs/modules/migrate/partials/nav.adoc:3]: ***

(to make it a section under the DMT section, not a new section at the same level)

**Review by rebekah-lawrence — CHANGES_REQUESTED** [review]

File: docs/modules/migrate/pages/dmt-command-reference.adoc (hunk 4 above)
Comment (kutluhanmetin) [reply docs/modules/migrate/pages/dmt-command-reference.adoc:41]: Actually that is not true, it is the opposite. If it is set migration is started without confirmation.

Comment (rebekah-lawrence) [reply docs/modules/migrate/pages/dmt-command-reference.adoc:41]: So, more like: If set, the migration starts immediately, without any confirmation.

File: docs/modules/migrate/partials/nav.adoc (hunk 8 above)
Comment (kutluhanmetin) [reply docs/modules/migrate/partials/nav.adoc]: https://github.com/hazelcast/hz-docs/pull/890/commits/7e48b6af17c9f43c94a9570a9da520a87ec82cf1

File: docs/modules/migrate/pages/dmt-command-reference.adoc (hunk 7 above)
Comment (kutluhanmetin) [reply docs/modules/migrate/pages/dmt-command-reference.adoc]: https://github.com/hazelcast/hz-docs/pull/890/commits/7e48b6af17c9f43c94a9570a9da520a87ec82cf1

File: docs/modules/migrate/pages/dmt-command-reference.adoc (hunk 6 above)
Comment (kutluhanmetin) [reply docs/modules/migrate/pages/dmt-command-reference.adoc]: https://github.com/hazelcast/hz-docs/pull/890/commits/7e48b6af17c9f43c94a9570a9da520a87ec82cf1

File: docs/modules/migrate/pages/dmt-command-reference.adoc (hunk 5 above)
Comment (kutluhanmetin) [reply docs/modules/migrate/pages/dmt-command-reference.adoc]: https://github.com/hazelcast/hz-docs/pull/890/commits/7e48b6af17c9f43c94a9570a9da520a87ec82cf1

File: docs/modules/migrate/pages/dmt-command-reference.adoc (hunk 4 above)
Comment (kutluhanmetin) [reply docs/modules/migrate/pages/dmt-command-reference.adoc]: https://github.com/hazelcast/hz-docs/pull/890/commits/7e48b6af17c9f43c94a9570a9da520a87ec82cf1

File: docs/modules/migrate/pages/dmt-command-reference.adoc (hunk 3 above)
Comment (kutluhanmetin) [reply docs/modules/migrate/pages/dmt-command-reference.adoc]: https://github.com/hazelcast/hz-docs/pull/890/commits/7e48b6af17c9f43c94a9570a9da520a87ec82cf1

File: docs/modules/migrate/pages/dmt-command-reference.adoc (hunk 2 above)
Comment (kutluhanmetin) [reply docs/modules/migrate/pages/dmt-command-reference.adoc]: https://github.com/hazelcast/hz-docs/pull/890/commits/7e48b6af17c9f43c94a9570a9da520a87ec82cf1

File: docs/modules/migrate/pages/dmt-command-reference.adoc (hunk 1 above)
Comment (kutluhanmetin) [reply docs/modules/migrate/pages/dmt-command-reference.adoc]: https://github.com/hazelcast/hz-docs/pull/890/commits/7e48b6af17c9f43c94a9570a9da520a87ec82cf1

**Review by rebekah-lawrence — APPROVED** [review]
LGTM

**Pull request merged**